class DataSourceAccessProblem(Exception):
    pass

class ViaAppiaException(Exception):
    pass